appveyor.yml

```diff
@@ -30,11 +30,6 @@ install:
   - conda env create --quiet --file "%ENV%"
   - activate "%ENV_NAME%"
   - conda install console_shortcut
-  # Fortran cells.
```

Comment:
We don't have any `Fortran` notebook yet and this does not work with latest Python. So I am removing this and in the future we can experiment with `msys2` instead.